stable/janusgraph/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for JanusGraph chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: gcr.io/cloud-solutions-images/janusgraph
+  tag: v1
+  pullPolicy: Always
+
+## The default configuration provided here uses attached storage for db and indexing
+## For a distributed deployment, increase the number of replicas and choose
+## a distributed backend for storage and indexing below (i.e. hbase and elasticsearch)
+replicaCount: 1
+
+## set any pod specific resource requests here
+resources: {}
+
+service:
+  type: ClusterIP  # Change to LoadBalancer if you plan to access JanusGraph outside k8s cluster
+  port: 8182
+  serviceAnnotations:
+    # the following line is ignored unless unless using a LoadBalancer with GCP
+    cloud.google.com/load-balancer-type: "Internal"
```

Comment:
Comment this out. It should be optional.

Reply:
fixed, thx